01week/bio/css/style.css

```diff
@@ -0,0 +1,97 @@
+* {
```

Comment:
I think it's better to use the body selector rather than the * selector. I believe I heard Kevin say this was part of best practices.